samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/AccountController.cs

```diff
@@ -40,6 +40,11 @@ public async Task<ActionResult> Login(string returnUrl)
                 IdpOptions = identityProvidersResponse.IdpOptions,  // You can keep IdpOptions unset (set to null) if you don't want or need social login buttons
             };
 
+            if (TempData.ContainsKey("TerminalStateMessage"))
```

Comment:
I thought this change was already included in master 🤔

Reply:
oh, it's in Master actually, need to update the branch